src/sentry/incidents/action_handlers.py

```diff
@@ -128,6 +128,24 @@ def send_alert(self, metric_value):
         pass
 
 
+@AlertRuleTriggerAction.register_type(
+    "pagerduty",
+    AlertRuleTriggerAction.Type.PAGERDUTY,
+    [AlertRuleTriggerAction.TargetType.OPTIONS],
+    integration_provider="pagerduty",
+)
+class PagerdutyActionHandler(ActionHandler):
```

Comment:
I note we name this `PagerDuty` (camelcase) in some places, and `Pagerduty` (titlecase) in others.

Reply:
Can we stick to camelcase?

Reply:
☝️

Reply:
good catch! I will keep it as PagerDuty everywhere.